tests/ssg_test_suite/common.py

```diff
@@ -628,14 +630,15 @@ def iterate_over_rules(product=None):
                     test_path = os.path.join(tests_dir, test_case)
                     if os.path.isdir(test_path):
                         continue
+                    # Filter out everything except the shell test scenarios.
+                    # Other files in rule directories are editor swap files
+                    # or other content than a test case.
+                    if not test_path.endswith(".sh"):
```

Comment:
Maybe it will be useful until we start ignoring it. Making it print a message for every skipped file will make it any auxiliary file we ship.
It also made me realize that non-dot-sh files also need to be included in the tarball.